src/Components/Common/UserDetails.tsx

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+
+function UserDetails(props: { children: React.ReactNode; title: string }) {
```

Comment:
Instead of passing label through children 
- Add `label` prop of type `string`

Reply:
`title` corresponds to `label` in the UserDetails component. I'm using the children prop for the data corresponding to that label like this : 

```typescript
<UserDetails title="District">
    <div className="font-semibold">
    	{user.district_object.name}
    </div>
<UserDetails>
```

Reply:
based on this @developedBySJ, is the change still needed?